code/game/machinery/doors/brigdoors.dm

```diff
@@ -210,6 +210,8 @@
 
 	. = TRUE
 
+	var/mob/M = usr
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Name it `user`.